maui-toc.html

```diff
@@ -723,6 +710,19 @@
             <li><a href="/maui/ImageEditor/accessibility">Accessibility</a></li>
         </ul>
     </li>
+    <li>
+        SfKanban
+        <ul>
+            <li><a href="/maui/kanban-board/overview">Overview</a></li>
+            <li><a href="/maui/kanban-board/Getting-Started">Getting Started</a></li>
+            <li><a href="/maui/kanban-board/migration">Migrate from Xamarin.Forms</a></li>
+            <li><a href="/maui/kanban-board/column">Column</a></li>
+            <li><a href="/maui/kanban-board/workflows">Workflows</a></li>
+            <li><a href="/maui/kanban-board/cards">Cards</a></li>
+            <li><a href="/maui/kanban-board/events">Events</a></li>
```

Comment:
@EzhilarasanElangovan31 change the file order, update Localization before Events

Reply:
changed the order